src/controllers/hotkeys/README.md

```diff
@@ -0,0 +1,44 @@
+# Custom Hotkeys
```

Comment:
should really move this to the chatterino wiki PR

Reply:
I think that documentation for developers is fine to stay within this repository. We also do similar thing with `src/providers/twitch/api/README.md`, which is just documentation about things that are important to developers only and have no meaning to end users.

Reply:
I thought of [this file](https://github.com/Chatterino/wiki/blob/master/docs/Contributing%20for%20Developers.md). Yeah looking at it now these docs don't fit there.